src/OnlineSales/Identity/AzureAdCookieEventsHandler.cs

```diff
@@ -0,0 +1,44 @@
+﻿// <copyright file="AzureAdCookieEventsHandler.cs" company="WavePoint Co. Ltd.">
+// Licensed under the MIT license. See LICENSE file in the samples root for full license information.
+// </copyright>
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authentication.Cookies;
+using Microsoft.AspNetCore.Identity;
+using OnlineSales.Entities;
+
+namespace OnlineSales.Identity;
+public class AzureAdCookieEventsHandler : CookieAuthenticationEvents
```

Comment:
@Exmirai the logic inside the AzureAdCookieEventsHandler and AzureAdJwtBearerEventsHandler is almost identical. Consider implementing the common part as a reusable block of code.